### ISSUE TITLE
Unable to run Godot from the editor

### ISSUE THREAD

**Issue by Shatur95** [habamax/vim-godot#9]
When I trying any command for running a have the following message:
![изображение](https://user-images.githubusercontent.com/22453358/83918089-b3a7ea00-a780-11ea-9980-72c71e192abb.png)

**Comment by Shatur95** [comment]
I will fix it myself, let me send a PR.

**Comment by habamax** [comment]
```
    elseif has("unix") " XXX: need test
        let cmd = "!xdg-open "
```

I don't have normal linux box so wasn't able to test it properly.

**Comment by Shatur95** [comment]
Yes, I read this comment already :)

**Comment by Shatur95** [comment]
Closed via #10
